how-to-docs.html

```diff
@@ -8,75 +8,87 @@
 
 <body>
 
-<p>Revised: 
-<!--webbot bot="Timestamp" S-Type="EDITED" S-Format="%d %B %Y" startspan -->24 March 2023<!--webbot bot="Timestamp" endspan i-checksum="39626" --></p>
+<p>Revised:
+2 November 2024</p>
 
 <h1>How To Instructions</h1>
-<p>These instruction assume you going to keep the issues lists source files in a directory named &quot;issues&quot;, 
-and the issues lists generated HTML files in a directory named &quot;issues-gh-pages&quot;.</p>
-<blockquote>
-<p>Keeping the source files and generated HTML files in separate directories 
-falls out from the GitHub automatic web page approach: files in a branch named &quot;gh-pages&quot; 
-are automatically published by GitHub as web pages.</p>
-</blockquote>
+<p>These instruction assume you going to keep the issues lists source files in a directory named &quot;issues&quot;.
+The issues are stored in XML files, one file per issue.
+The software in the <code>src</code> directory builds programs
+that read the XML files and generates the HTML issues lists.
+When changes to the XML files are pushed to the GitHub repository an automated
+<a href="https://docs.github.com/en/actions">GitHub Actions</a> workflow
+generates new HTML files and commits them to the <code>gh-pages</code> branch.
+The content of the <code>gh-pages</code> branch is automatically deployed
+to <a href="https://cplusplus.github.io/LWG/">https://cplusplus.github.io/LWG/</a>.
+</p>
 
 <h2>Prerequisites</h2>
 <ul>
   <li>Git</li>
   <li>A C++17 compiler. Clang 9 works. GCC 8.x works for unix-like systems, but GCC 9.3 is recommended (because its <code>std::filesystem</code> supports Windows, and no additional <code>-lstdc++fs</code> linker option is needed).</li>
   <li>Windows is a prerequisite to use the Windows <code>.bat</code> scripts, or
```

Comment:
These substatements are in fact both true.
```suggestion
  <li>Windows is a prerequisite to use the Windows <code>.bat</code> scripts;
```